recipes/ogre/1.x/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(KEEP_RPATHS)
+
+
+find_package(OGRE ${OGRE_VERSION} COMPONENTS OgreMain REQUIRED)
+add_executable(ogre_main ogre_main.cpp)
+target_link_libraries(ogre_main OGRE::OgreMain)
+
+set_target_properties(ogre_main PROPERTIES
+CXX_STANDARD 17
```

Comment:
Who needs C++17? Ogre? If yes, need to check cppstd + compiler version. Otherwise, you can down it to C++11

Reply:
will look at it. I think C++11 should be sufficient

Reply:
```suggestion
CXX_STANDARD 11
```

Reply:
done